packages/sanity/src/core/form/inputs/DateInputs/base/DateTimeInput.tsx

```diff
@@ -48,6 +49,7 @@ export const DateTimeInput = forwardRef(function DateTimeInput(
   const [popoverRef, setPopoverRef] = useState<HTMLElement | null>(null)
   const ref = useRef<HTMLInputElement | null>(null)
   const buttonRef = useRef(null)
+  const {t} = useTranslation()
```

Comment:
Instead of using `useTranslation` directly here, could you add a new key to `calendarLabels` and have it be passed in? That way we can keep this component free from the dependency to useTranslation, and it also aligns better with how this component already is localized